privacyidea/api/lib/utils.py

```diff
@@ -267,3 +268,18 @@ def verify_auth_token(auth_token, required_role=None):
                         "this resource!").format(required_role),
                         id=ERROR.AUTHENTICATE_MISSING_RIGHT)
     return r
+
+
+def check_policy_name(name):
+    """
+    This function checks, if the given name is a valid policy name.
+
+    :param name: The name of the policy
+    :return: Raises a ParameterError in case of an invalid name
+    """
+    if not re.match('^[a-zA-Z0-9_.\- ]*$', name):
+        raise ParameterError(_("The name of the policy may only contain "
+                               "the characters a-zA-Z0-9_.- "))
+
+    if name.lower() == "check":
+        raise ParameterError(_("T'check' is an invalid policy name."))
```

Comment:
maybe we can define a list of invalid names (there are a few IIRC).
And is this `T` in the error message necessary? It looks a bit like [Klingon](https://en.wikipedia.org/wiki/Klingon_language).

Reply:
check this out. 
0ea6ecfd